common/net/key_storage.go

```diff
@@ -0,0 +1,112 @@
+package net
```

Comment:
this package `net` is not suitable for such features (encrypt/decrypt), I think it should be placed in `common/crypto`

Reply:
thx